lib/src/components/NxNumberInput/stateHelpers.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import { StateProps, Validator } from './types';
+import { curryN, trim } from 'ramda';
+
```

Comment:
Are these identical to the `NxTextInput` state helpers? If so, why duplicate them?

That said, I have a suggestion which would make them non-identical.  Just like `trimmedValue` is provided as a convenience in addition to the raw `value`, I think for number inputs we should have a `numberValue` property that exposes the value parsed to a `number` as opposed to as a string.

Reply:
> Are these identical to the `NxTextInput` state helpers? If so, why duplicate them?
> 
> That said, I have a suggestion which would make them non-identical. Just like `trimmedValue` is provided as a convenience in addition to the raw `value`, I think for number inputs we should have a `numberValue` property that exposes the value parsed to a `number` as opposed to as a string.

The file was duplicated, but the component actually references the file in `/NxTextInput/`, removed the dupe file here: https://github.com/sonatype/sonatype-react-shared-components/pull/486/commits/a30cce6c4eba2979a2669e8efdf08641bc4e9527